src/google-maps/google-map/google-map.spec.ts

```diff
@@ -280,6 +285,23 @@ describe('GoogleMap', () => {
     expect(addSpy).toHaveBeenCalledWith('projection_changed', jasmine.any(Function));
     subscription.unsubscribe();
   });
+
+  it('should set the map type', () => {
+    mapSpy = createMapSpy(DEFAULT_OPTIONS);
+    mapConstructorSpy = createMapConstructorSpy(mapSpy).and.callThrough();
+
+    const fixture = TestBed.createComponent(TestApp);
+    fixture.componentInstance.mapTypeId = 'terrain' as unknown as google.maps.MapTypeId;
```

Comment:
Why not google.maps.MapTypeId.TERRAIN

Reply:
Because the global `google` variable isn't available when running unit tests.